README.md

```diff
@@ -147,6 +147,49 @@ matched), unless the `emailDomainMustHaveDot` option is disabled.
 Use `LinkType.EMAIL` for this, and see [test cases
 here](src/test/java/org/nibor/autolink/AutolinkEmailTest.java).
 
+### Hash tag extraction
+Extract Twitter/Facebook/Instagram Hash tags such as #photooftheday. Hash tags need to start with `#` and ends with a
+space. In general hash tags do not contain special characters other than `_`. Also hash tags do not start with a
+number. For example, #123 is not a valid hash tag.
+
+Examples:
+
+* `#photooftheday` → [#photooftheday]()
+* `#photo_of_the_day.` → [#photo_of_the_day]().
+
+You can configure the allowed special characters in the hash tag. For example if you want to extract the hash tags
+containing both `_` and '-', then you can configure your builder as shown:
```

Comment:
What's the reason for allowing this to be configured? Is there a service that allows `-` in hashtags? If not, I'd prefer to keep it strict and minimal for now.